routes/likeRouter.js

```diff
@@ -0,0 +1,12 @@
+const { verifyJWT } = require("../middleware/auth");
+const express = require("express");
+const likeController = require("../controllers/likeController");
+
+const router = express.Router();
+
+router.post("/:productId", verifyJWT, likeController.createLike);
+router.delete("/delete/:productId", verifyJWT, likeController.deleteLike);
```

Comment:
```suggestion
router.delete("/:productId", verifyJWT, likeController.deleteLike);
```
이미 delete 라는 HTTP method에 따라 자명하게 구분되고 있기도 하고, REST에서는 엔드포인트에 동사를 작성하지 않기 때문에 지워주셔도 됩니다.